code/game/objects/items/weapons/storage/fancy/cigar.dm

```diff
@@ -15,11 +15,7 @@
 	storage = /datum/storage/box/cigar
 
 /obj/item/box/fancy/cigar/Initialize(ml, material_key)
-	. = ..()
-	initialize_reagents()
-
-/obj/item/box/fancy/cigar/initialize_reagents(populate)
-	create_reagents(10 * max(1, storage?.storage_slots))
+	chem_volume = 10 * max(1, /datum/storage/box/cigar::storage_slots)
```

Comment:
i think this won't work for subtypes that use a larger or smaller storage subtype, idk if that's a concern. before `..()` in Initialize `storage`, if non-null, is a path, so we might be able to just do `storage ? storage::storage_slots : null` with a comment explaining `storage` is a path or null pre-init

Reply:
`storage` is a datum at this point, it's initialized in `New()`, I just didn't think it mattered as no subtypes change `storage`.